README.md

```diff
@@ -147,7 +147,7 @@ restart it. No worries; it's pretty quick.
 
 ### End-to-end (E2E) Tests
 
-E2E tests are in the `e2e` directory, side by side with the `app` folder.
+E2E tests are in the `e2e` directory, side by side with the `src` folder.
```

Comment:
The paragraph says they are in the `e2e` folder, and that it is a sibling folder to `src`. Did you misread or am I missing something?